src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/xunit.runner.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "parallelAlgorithm": "conservative",
+  "parallelizeTestCollections": false,
```

Comment:
Did this impact test suite duration much?

Reply:
This was another attempt of mine to resolve the issue by tweaking xunit settings. It turned out these changes didn't solve the issue. I'll remove this `xunit.runner.json` file. Another :facepalm: